http4s/src/main/scala/com/ccadllc/cedi/dtrace/http4s/server/server.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Comcast Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ccadllc.cedi.dtrace
+package http4s
+
+import cats.effect.Sync
+import cats.implicits._
+
+import org.http4s.{ Header => H4sHeader, _ }
+import org.http4s.parser.HttpHeaderParser
+import org.http4s.util.CaseInsensitiveString
+
+import scala.language.higherKinds
+
+package object server {
+  /**
+   * This function drops the parser for the built-in `http4s` `X-B3-Trace-Id` header since it only successfully parses 64 bit
+   * values, failing on 128 bit values and with cedi dtrace, the 128 bit value will be produced and are preferred
+   * to consume in order to represent traces uniquely in a distributed environment.  Once `http4s` updates the parser to handle
+   * 128 bit values, this can be removed.  This should be invoked in your initialization code.
```

Comment:
Can we add a link to the http4s issue?

Reply:
I'll see if there is one / add one if there isn't - this was an exchange between Ross and I on gitter.